Source/WebCore/page/CaptionUserPreferencesMediaAF.cpp

```diff
@@ -370,7 +369,7 @@ Color CaptionUserPreferencesMediaAF::captionsTextColor(bool& important) const
     RetainPtr color = adoptCF(MACaptionAppearanceCopyForegroundColor(kMACaptionAppearanceDomainUser, &behavior)).get();
     Color textColor(roundAndClampToSRGBALossy(color.get()));
     if (!textColor.isValid()) {
-        // This default value must be the same as the one specified in mediaControls.css for -webkit-media-text-track-container.
+        // FIXME: This is potentially wrong since modern media controls happened.
```

Comment:
This comment is vague. Nobody will remember the name "modern media controls" in the future. Likely would be better to quote the year we changed the media controls implementation on most platforms or something like that. Put yourself in the shoes of the future person reading the comment.

But also, maybe you could quickly try to assess if this is wrong or not, instead of just reflecting that it’s no longer relevant whether it matches an unused CSS source file?